spa/src/components/portal/ContributionsList/ContributionDetail/BillingDetails/BillingDetails.styled.ts

```diff
@@ -1,6 +1,24 @@
+import { TextField } from 'components/base';
 import styled from 'styled-components';
 
 export const CheckboxLabel = styled.span`
   color: ${({ theme }) => theme.basePalette.greyscale.black};
   font-size: ${({ theme }) => theme.fontSizesUpdated.sm};
 `;
+
+export const StyledTextField = styled(TextField)`
```

Comment:
```suggestion
export const AmountField = styled(TextField)`
```
Clearer about its purpose (e.g. offsetting the adornment)

Reply:
[7651576](https://github.com/newsrevenuehub/rev-engine/pull/1625/commits/76515762ab213924e73bdbc66a8be458453a81fb)